I'm not getting this to build under ARMC6. User error?
```
mbed compile -t arm --profile debug  
[mbed] Working path "/home/dan/Mbed Programs/ng-robot-firmware" (program)
Building project ng-robot-firmware (NUCLEO_H743ZI2, ARMC6)
Scan: ng-robot-firmware
Using ROM region application in this build.
  Region application: size 0x200000, offset 0x8000000
Using RAM region application_ram in this build.
  Region application_ram: size 0x80000, offset 0x24000000
Compile [ 41.7%]: mbed_printf_armlink_overrides.c
Compile [ 42.0%]: mbed_printf_implementation.c
Compile [ 42.3%]: mbed_stats.c
Compile [ 42.6%]: mbed_wait_api_no_rtos.c
Compile [ 42.9%]: mbed_sdk_boot.c
Compile [ 43.2%]: mbed_semihost_api.c
Compile [ 43.5%]: mbed_printf_wrapper.c
Compile [ 43.8%]: mbed_error.c
[Error] mbed_error.c@522,41: use of undeclared identifier 'Image$$ARM_LIB_STACK$$ZI$$Limit'
[Error] mbed_error.c@522,41: use of undeclared identifier 'Image$$ARM_LIB_STACK$$ZI$$Limit'
[ERROR] ./mbed-os/platform/source/mbed_error.c:522:41: error: use of undeclared identifier 'Image$$ARM_LIB_STACK$$ZI$$Limit'
        uint32_t msp_size = MAX(0, (int)__INITIAL_SP - (int)msp_sp);
                                        ^
./mbed-os/cmsis/CMSIS_5/CMSIS/TARGET_CORTEX_M/Include/cmsis_armclang.h:124:35: note: expanded from macro '__INITIAL_SP'
#define __INITIAL_SP              Image$$ARM_LIB_STACK$$ZI$$Limit
                                  ^
./mbed-os/platform/source/mbed_error.c:522:41: error: use of undeclared identifier 'Image$$ARM_LIB_STACK$$ZI$$Limit'
./mbed-os/cmsis/CMSIS_5/CMSIS/TARGET_CORTEX_M/Include/cmsis_armclang.h:124:35: note: expanded from macro '__INITIAL_SP'
#define __INITIAL_SP              Image$$ARM_LIB_STACK$$ZI$$Limit
                                  ^
2 errors generated.

[mbed] ERROR: "/home/dan/.config/Mbed Studio/mbed-studio-tools/python/bin/python" returned error.
       Code: 1
       Path: "/home/dan/Mbed Programs/ng-robot-firmware"
       Command: "/home/dan/.config/Mbed Studio/mbed-studio-tools/python/bin/python -u /home/dan/Mbed Programs/ng-robot-firmware/mbed-os/tools/make.py -t arm -m NUCLEO_H743ZI2 --profile debug --source . --build ./BUILD/NUCLEO_H743ZI2/ARM-DEBUG"
       Tip: You could retry the last command with "-v" flag for verbose output
---
```

--- a/platform/source/mbed_error.c
+++ b/platform/source/mbed_error.c
@@ -67,6 +67,8 @@ static mbed_error_status_t handle_error(mbed_error_status_t error_status, unsign
 static bool is_reboot_error_valid = false;
 #endif
 
+extern uint32_t __INITIAL_SP;
+
 //Helper function to halt the system
 static MBED_NORETURN void mbed_halt_system(void)
 {

